.travis.yml

```diff
@@ -9,39 +9,47 @@ matrix:
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
+        - USE_CGAL=1
+        - USE_VTK=1
     - os: osx
       compiler: clang
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
         - BLASLAPACK_IMPLEMENTATION=vecLib
+        - BUILD_DOCUMENTATION=1
     - os: osx
       compiler: clang
       env:
         - USE_PROJECT=0
-        - USE_SYSTEM=0
         - BLASLAPACK_IMPLEMENTATION=Auto
-    - os: osx
+    - os: linux
```

Comment:
Yes default case is Auto and when nothing is specified it is Auto.
Auto is (as written in UseAuto.cmake):
    if (WIN32)
        priorities(MKL OpenBLAS LAPACK)
    elseif (APPLE)
        priorities(MKL OpenBLAS vecLib LAPACK)
    else()
        priorities(MKL OpenBLAS Atlas LAPACK)
    endif()
So Yes we test on osX once with vecLib, and twice with OpenBLAS (as the MKL is not present).
But, the Auto, and void option for BLASLAPACK_IMPLEMENTATION has to be tested anyway.